### PR TITLE
14-upload-a-file-for-submission

### PR DESCRIPTION
**Pull request by yashsavani19**
Accepts Bibtex fornat for submission, and automatically parses the information into the field.

### CODE REVIEW

File: frontend/src/app/moderator/page.tsx
Comment (AnnetteScott) [inline frontend/src/app/moderator/page.tsx:1]: Why?

**Review by AnnetteScott — CHANGES_REQUESTED** [review]
Again, per our documentation, you need to request a review from everyone on the team when you create the PR

**Review by AnnetteScott — CHANGES_REQUESTED** [review]
These changes do not align with the requirements for the user story

**Review by yashsavani19 — COMMENTED** [review]
Fixed the changes required. Now the information shows under suggest article page

**Review by AnnetteScott — APPROVED** [review]

**Pull request merged**